### PR TITLE
Fix/indent with alignment

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/common/web/statics/styles/draftDefault.rtlignore.scss [hunk 1]
```diff
@@ -14,14 +14,14 @@
   }
   @mixin indent_blocks($depth, $indent) {
     .public-DraftStyleDefault-block-depth#{$depth} {
-    &.public-DraftStyleDefault-ltr, & .public-DraftStyleDefault-ltr{
+    &.public-DraftStyleDefault-text-ltr, & .public-DraftStyleDefault-text-ltr{
```
Comment (shaulgo) [inline packages/common/web/statics/styles/draftDefault.rtlignore.scss:17]: check if both of these cases are needed

File: packages/common/web/statics/styles/draftDefault.rtlignore.scss [hunk 2]
```diff
@@ -14,14 +14,14 @@
...
        /*!rtl:begin:ignore*/
       margin-left: #{$indent}em !important;
       /*!rtl:end:ignore*/
     }}
   
     .public-DraftStyleDefault-block-depth#{$depth} {
-    &.public-DraftStyleDefault-rtl, & .public-DraftStyleDefault-rtl{
+    &.public-DraftStyleDefault-text-rtl, & .public-DraftStyleDefault-text-rtl{
```
Comment (shaulgo) [inline packages/common/web/statics/styles/draftDefault.rtlignore.scss:24]: check if both of these cases are needed

Comment (maimonav) [reply packages/common/web/statics/styles/draftDefault.rtlignore.scss:24]: fixed, nested `& .public-DraftStyleDefault-text-rtl` is not needed.

File: packages/common/web/statics/styles/draftDefault.rtlignore.scss (hunk 1 above)
Comment (maimonav) [reply packages/common/web/statics/styles/draftDefault.rtlignore.scss:17]: fixed, nested & .public-DraftStyleDefault-text-rtl is not needed.

**Pull request merged**